src/quo2/components/inputs/recovery_phrase/view.cljs

```diff
@@ -31,7 +31,8 @@
         set-default #(reset! state :default)]
     (fn [{:keys [customization-color override-theme blur? on-focus on-blur mark-errors?
                  error-pred word-limit]
-          :or   {customization-color :blue}
+          :or   {customization-color :blue
+                 error-pred          (constantly false)}
```

Comment:
what is `pred`?

Reply:
`pred` is an abbreviation for *predicate* that's used a lot in Clojure. Details here https://guide.clojure.style/#idiomatic-names

Reply:
@ilmotta @J-Son89 
 Hmm... `error-pred` receives a function that will behave as a predicate to validate words, if the predicate applied to a word is false, then the word is shown in red inside the input.

So if I name it as `error?`, I think it'd look as if we expect a boolean. I'd prefer to keep `error-pred` since IMO is better than `error?`, wdyt?